monai/bundle/scripts.py

```diff
@@ -142,6 +144,30 @@ def _download_from_github(repo: str, download_path: Path, filename: str, progres
     extractall(filepath=filepath, output_dir=download_path, has_base=True)
 
 
+def _download_from_ngc(
+    download_path: Path, filename: str, version: str, model_prefix: str = "nvidia/monaitoolkit/", progress: bool = True
+):
+    url = _get_ngc_url(model_name=filename, version=version, model_prefix=model_prefix)
```

Comment:
The filename of the same Github and NGC model is different to each other. It would be important to add a check if the NGC model name has a "monai_" in the beginning, and add it if not present